ommp:
  hardware_interface:
    loop_hz: 20
    joints:
      - front_left_wheel_joint
      - front_right_wheel_joint
      - camera_dummy_joint
      - back_left_wheel_joint
      - back_right_wheel_joint
      - joint_0
      - joint_1
      - joint_2
      - joint_3
      - joint_4
      - joint_5
      - joint_finger1
      - joint_finger2

    initial_states:
      front_left_wheel_joint: 0.0
      front_right_wheel_joint: 0.0
      camera_dummy_joint: 0.0
      back_left_wheel_joint: 0.0
      back_right_wheel_joint: 0.0
      joint_0: 0.0
      joint_1: -1.57
      joint_2: 2.61
      joint_3: 0.0
      joint_4: -1.57
      joint_5: 0.0
      joint_finger1: 0.0
      joint_finger2: 0.0


